azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/configuration/events/CommunicationUIRemoteParticipantJoinedEvent.java

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.android.communication.ui.configuration.events;
+
+import com.azure.android.communication.common.CommunicationIdentifier;
+
+import java.util.List;
+
+/**
+ * Remote participants joined event with communication identifiers.
+ */
+public final class CommunicationUIRemoteParticipantJoinedEvent {
+    private final List<CommunicationIdentifier> identifiers;
+
+    /**
+     * Create {@link CommunicationUIRemoteParticipantJoinedEvent} with user identifiers.
+     *
+     * @param identifiers {@link CommunicationIdentifier};
+     */
+    public CommunicationUIRemoteParticipantJoinedEvent(final List<CommunicationIdentifier> identifiers) {
```

Comment:
I think we should use here readonly collection instead of List. Same in the getter